pkgs/development/python-modules/pytest-repeat/default.nix

```diff
@@ -0,0 +1,30 @@
+{ lib
+, buildPythonPackage
+, fetchPypi
+, setuptools_scm
+, pytest
+}:
+
+buildPythonPackage rec {
+  pname = "pytest-repeat";
+  version = "0.6.0";
+
+  src = fetchPypi {
+    inherit pname version;
+    sha256 = "84aba2fcca5dc2f32ae626a01708f469f17b3384ec3d1f507698077f274909d6";
+  };
+
+  buildInputs = [ setuptools_scm ];
+  propagatedBuildInputs = [ pytest ];
```

Comment:
Don't propagate pytest, it's a special case. Plugins should only use it for build time of itself.